src/components/CFloatingButton/index.meta.js

```diff
@@ -0,0 +1,108 @@
+import { merge } from 'lodash';
+import CButton from '../CButton/index.meta';
+
+export default {
+  group: 'actions',
+  type: 'floating-button',
+  name: 'Floating Button',
+  icon: 'control_point',
+  children: [
+    'floating-button-item',
+  ],
+  actions: merge({}, CButton.actions),
+  events: merge({}, CButton.events),
+  options: {
+    color: true,
+    theme: true,
+    itemsCount: {
+      type: 'childrenCountInput',
+      name: 'Number of Child Elements',
+      value: '0',
+      validation: {
+        required: true,
+        min: 0,
+      },
+      priority: 3,
+    },
+    openOnHover: {
+      type: 'check',
+      name: 'Open On Hover',
+      value: true,
+      priority: 4,
+    },
+    absolute: {
+      type: 'check',
+      name: 'Position The Element Absolutely',
+      value: false,
+      priority: 5,
+    },
+    fixed: {
+      type: 'check',
+      name: 'Position The Element Fixed',
```

Comment:
This should have more Veka-friendly name, same as absolute prop. 
How about: `Float in container` and `Float on page`? 🤔